lib/diagnostic.js

```diff
@@ -32,6 +44,32 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update(
```

Comment:
This will try to find documents with the matching `members` key and value push the value of `House` into the document's `House` key value as if it were an array.  Since there aren't any documents with a `members` key, none of these queries will find anything.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```